net-fs/curlftpfs/curlftpfs-0.9.2.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="Filesystem for accessing FTP hosts"
+DESCRIPTION="A file system for accessing ftp hosts based on FUSE and \
+libcurl.
+As the FTP protocol is not very feature rich, this filesystem does not \
+fulfill every constraint of a real filesystem, but it should be usable \
+for simple tasks like copying and editing files.
+
+CurlFtpFS differentiates itself from other FTP filesystems because it \
+features:
+
+* SSLv3 and TLSv1 support
+* connecting through tunneling HTTP proxies
+* automatically reconnection if the server times out
+* transform absolute symlinks to point back into the ftp file system
+
+Usage on Haiku: First start the userland_fs server:
+	/system/servers/userlandfs_server curlftpfs &
```

Comment:
I've moved the "instructions" to a readme.txt in $docDir and added a small introduction for it. Please let me if that's OK and if anything can be improved (e.g. english style or grammar).
I've also converted the INSTALL to use **`install`** instead of **`mkdir`** and **`cp`**. It works fine, but I don't know if everyone prefers the "new" INSTALL ;-)
I think the recipe is good enough for being merged. IMO the readme.txt is both usable and accurate regarding the WIP or "workaround" status of the launch method.